src/MCPServer/lib/assets/workflow.json

```diff
@@ -2665,6 +2665,38 @@
                 "pt_BR": "Carregar etiquetas de metadata/file_labels.csv",
                 "sv": "Hämta märkning från metadata/file_labels.csv"
             },
+            "exit_codes": {
+                "0": {
+                    "job_status": "Completed successfully",
+                    "link_id": "657b0b54-dc65-4e6c-bc90-ef0e3e753128"
+                }
+            },
+            "fallback_job_status": "Failed",
+            "fallback_link_id": "657b0b54-dc65-4e6c-bc90-ef0e3e753128",
+            "group": {
+                "en": "Characterize and extract metadata",
+                "es": "Caracterizar y extraer metadatos",
+                "fr": "Caractériser et extraire les métadonnées",
+                "ja": "メタデータの特徴付けと抽出",
+                "pt_BR": "Caracterizar e extrair metadados",
+                "sv": "Karaktärisera och extrahera metadata"
+            }
+        },
+        "657b0b54-dc65-4e6c-bc90-ef0e3e753128": {
```

Comment:
Keys are currently sorted. I suggest to move this item between `653b134f-4a37-4578-a286-7f2072e89f9e` and `66c9c178-2224-41c6-9c0b-dcb60ff57b1a` just so diffs may be clearer in the future if we have new dumps generated for other reasons.

Reply:
Thanks, I didn't realize that. Moved now.